app/controllers/users/unlocks_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::UnlocksController < Devise::UnlocksController
```

Comment:
I'm concerned that having lots of files filled with comments makes the whole projects more burdensome to navigate - if we really must have these files for devise to operate that's one thing, but these should all exist in the gem - can't we add them individually as needed?